auth_oauth_multi_token/models/res_users.py

```diff
@@ -24,13 +20,15 @@ def _generate_oauth_master_uuid(self):
         readonly=True,
         groups="base.group_system",
     )
+
     oauth_access_max_token = fields.Integer(
         string="Max Number of Simultaneous Connections", default=10, required=True
     )
-    oauth_master_uuid = fields.Char(
+
+    # use the oauth_access_token field as oauth_master_uuid
+    oauth_access_token = fields.Char(
         string="Master UUID",
```

Comment:
You only need to define the default, isn't it?

Reply:
The origninal definition is this:
`
oauth_access_token = fields.Char(string='OAuth Access Token', readonly=True, copy=False, prefetch=False)
`

Reply:
I am also not very happy for reusing this field. We could leave the string value as is and just add an comment that this field is used as Master UUID instead of access_token.
The required=True I just took from the oauth_master_uuid field, but I am not sure if this is needed to force the default creation? It shouldn't be empty or the same value as another user, to make sure that the access_token is not used by a different user.